repos/spack_repo/builtin/packages/json_glib/package.py

```diff
@@ -61,3 +64,13 @@ def build(self, spec, prefix):
     def install(self, spec, prefix):
         """Run the AutotoolsPackage install phase"""
         make("install")
+
+    def url_for_version(self, version):
+        return f"https://download.gnome.org/sources/json-glib/{version.up_to(2)}/json-glib-{version}.tar.xz"
+
+
+class MesonBuilder(meson.MesonBuilder):
+    @when("@1.9.2:")
+    def meson_args(self):
+        args = ["-Ddocumentation=disabled", "-Dman=false", "-Dtests=false", "-Dconformance=false"]
```

Comment:
I only needed to disable `documentation` to get the newer versions to build. Any particular reason for disabling man pages and the tests as well?

Reply:
> I only needed to disable `documentation` to get the newer versions to build. Any particular reason for disabling man pages and the tests as well?

I can remove those extra configure arguments.  I was just trying to minimize all extra dependencies.